atd-vze/src/containers/DefaultLayout/DefaultHeader.js

```diff
@@ -79,26 +79,28 @@ const DefaultHeader = props => {
           </DropdownMenu>
         </UncontrolledDropdown>
         <UncontrolledDropdown nav direction="down">
-            <DropdownToggle nav>
-              <i className="fa fa-question-circle fa-2x" />
-            </DropdownToggle>
-            <DropdownMenu right>
-              <DropdownItem header tag="div" className="text-center">
-                <strong>Help</strong>
-              </DropdownItem>
-              <DropdownItem
-                href="https://atd.knack.com/dts#new-service-request/?view_249_vars=%7B%22field_398%22%3A%22Bug%20Report%20%E2%80%94%20Something%20is%20not%20working%22%2C%22field_399%22%3A%22Vision%20Zero%20(Editor)%22%7D"
-                target="_blank"
-              >
-                <i class="fa fa-bug" /> Report a bug&nbsp;&nbsp; <i className="fa fa-external-link" />
-              </DropdownItem>
-              <DropdownItem
-                href="https://atd.knack.com/dts#new-service-request/?view_249_vars=%7B%22field_398%22%3A%22Feature%20or%20Enhancement%20%E2%80%94%20An%20application%20I%20use%20could%20be%20improved%22%2C%22field_399%22%3A%22Vision%20Zero%20(Editor)%22%7D"
-                target="_blank"
-              >
-                <i class="fa fa-wrench" /> Request an enhancement&nbsp;&nbsp; <i className="fa fa-external-link" />
-              </DropdownItem>
-            </DropdownMenu>
+          <DropdownToggle nav>
+            <i className="fa fa-question-circle fa-2x" />
+          </DropdownToggle>
+          <DropdownMenu right>
+            <DropdownItem header tag="div" className="text-center">
+              <strong>Help</strong>
+            </DropdownItem>
+            <DropdownItem
+              href="https://atd.knack.com/dts#new-service-request/?view_249_vars=%7B%22field_398%22%3A%22Bug%20Report%20%E2%80%94%20Something%20is%20not%20working%22%2C%22field_399%22%3A%22Vision%20Zero%20(Editor)%22%7D"
+              target="_blank"
+            >
+              <i className="fa fa-bug" /> Report a bug&nbsp;&nbsp;{" "}
+              <i className="fa fa-external-link" />
+            </DropdownItem>
+            <DropdownItem
+              href="https://atd.knack.com/dts#new-service-request/?view_249_vars=%7B%22field_398%22%3A%22Feature%20or%20Enhancement%20%E2%80%94%20An%20application%20I%20use%20could%20be%20improved%22%2C%22field_399%22%3A%22Vision%20Zero%20(Editor)%22%7D"
+              target="_blank"
+            >
+              <i className="fa fa-wrench" /> Request an enhancement&nbsp;&nbsp;{" "}
+              <i className="fa fa-external-link" />
+            </DropdownItem>
+          </DropdownMenu>
```

Comment:
this was formatting with prettier 💅 and changing instances of `class` to `className` because the console was throwing an error